samza-yarn/src/main/java/org/apache/samza/util/hadoop/CertFSException.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.util.hadoop;
+
+public class CertFSException extends Exception {
+
+  /**
+   * Constructs an {@code CertFSException} with {@code null}
+   * as its error detail message.
+   */
+  public CertFSException() {
+    super();
```

Comment:
I think this is a common way to define a type of exception class. We may choose not to use it but we should have this option.